### PR TITLE
Add Android ADB Input extension

### PR DESCRIPTION
**Pull request by STAR-ZERO**
## Description

Run Android [ADB](https://developer.android.com/studio/command-line/adb) command `adb shell input text {text}` to enter text on your Android device.

## Screencast

https://user-images.githubusercontent.com/376376/155133342-ce2f81e2-67cb-406d-85fc-cad4826ba452.mov

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)

### CODE REVIEW

File: extensions/android-adb-input/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    }
+
+    (async () => {
+      const toast = await showToast({
+        title: "Search devices...",
+        style: Toast.Style.Animated,
+      });
+
+      try {
+        const devices = await getDeives();
```
Comment (pernielsentikaer) [inline extensions/android-adb-input/src/index.tsx:116]:
```suggestion
        const devices = await getDevices();
```
Totally NIT but could be nice to have corrected 😄

File: extensions/android-adb-input/src/index.tsx [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+  const home = process.env.HOME;
+  return `${home}/Library/Android/sdk/platform-tools`;
+}
+
+function existsAdb(): boolean {
+  return fs.existsSync(`${getAdbDir()}/adb`);
+}
+
+async function getDeives(): Promise<string[]> {
```
Comment (pernielsentikaer) [inline extensions/android-adb-input/src/index.tsx:37]:
```suggestion
async function getDevices(): Promise<string[]> {
```
Totally NIT but could be nice to have corrected 😄

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @STAR-ZERO 👋 

Awesome work, seems to do the job as it should, and nice fallback if anything goes wrong. Only left two small NITs for you to correct, otherwise awesome 🔥 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

File: extensions/android-adb-input/src/index.tsx (hunk 1 above)
Comment (STAR-ZERO) [reply extensions/android-adb-input/src/index.tsx]: Fix [a5f1062](https://github.com/raycast/extensions/pull/964/commits/a5f106229b7eee34278b237abe5ce7f24fd4aa8b)

File: extensions/android-adb-input/src/index.tsx (hunk 2 above)
Comment (STAR-ZERO) [reply extensions/android-adb-input/src/index.tsx]: Fix [a5f1062](https://github.com/raycast/extensions/pull/964/commits/a5f106229b7eee34278b237abe5ce7f24fd4aa8b)

**Review by pernielsentikaer — APPROVED** [review]
Hi @STAR-ZERO 👋 

Thanks for the update, let's get it to the final stage 💯 🚀

**Pull request merged**